### ISSUE TITLE
About the new handling of filters in v22

### ISSUE THREAD

**Issue by haverholm** [tuskyapp/Tusky#3746]
I'm really disappointed to see that Tusky has started aping Mastodon's approach to filtered toots, where a disruptive message  in your timeline asks if maybe you want to see the message anyway. I assume most people don't since they actively filtered a term to begin with.

Until the latest version, filtered toots just... didn't show. Perfection. Now, to avoid the annoying "SHOW ANYWAY" flags, I am expected to edit each of my filters and select "hide completely" to switch back to the former default.

Now, I have a lot of filters, between 20 and 70 on each of my four accounts. Did it occur to the developers that a) changing the default display settings for terms *that users already opted out of* was a bad idea, and b) maybe it shouldn't be up to users to revert your brain fart on a granular, per filter level?

**Tl;dr**
1. Change the default display setting for filtered toots back to "hide completely".
2. Allow users to change display settings for all filters rather than for individual terms.

* * * *
- Tusky Version: 22

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
Yes this is a very unfortunate situation, mostly because of Mastodon changing their Api and us not being aware of the implications this change has. We will definitely take better care next time.
The filter implementation in Tusky 22 is now correct, and we decided to leave it that way and not make it configurable in order to be consistent with Mastodon and not bloat our code. Sorry for the inconvenience.
